beacon-light-client/plonky2/light_cleaner_common.ts

```diff
@@ -33,8 +33,8 @@ export async function lightClean(this: any, db: Redis, prefix: any) {
   }
 }
 
-export function getOptions() {
-  return yargs(hideBin(process.argv))
+export function getOptions(balanceVerification: boolean) {
+  let options = yargs(hideBin(process.argv))
```

Comment:
Delete this function and use the cmdline options builder instead. I find the boolean parameter unintuitive